arangod/StorageEngine/TransactionState.cpp

```diff
@@ -202,6 +202,24 @@ void TransactionState::allCollections(                     // iterate
     }
   }
 }
+  
+/// @brief whether or not the collection is used in the transaction
+bool TransactionState::containsCollection(TRI_voc_cid_t cid, AccessMode::Type accessType) {
```

Comment:
I think in a running transaction we should be able to check the transaction name

Reply:
I have not the slightest idea what a transaction name is.